docs-new/docs/contribute/general/refinement-guide.md

```diff
@@ -0,0 +1,52 @@
+# Refinement Timeblock
+
+During the Refinement Timeblock, maintainers engage in focused technical
+discussions on open issues and pull requests.
+This dedicated time allows for in-depth conversations, knowledge sharing,
+and collective decision-making.
+It is an opportunity for the team to synchronize their understanding of
+ongoing developments, address challenges, and ensure a common vision for the project.
+
+**Purpose and Goals:**
+
+- **Alignment:** Ensure a shared understanding among maintainers regarding ongoing
+developments, project goals, and upcoming tickets.
+- **Technical Depth:** Delve into the technical aspects of open issues and pull
+requests, facilitating a deeper understanding of proposed Pull requests.
+- **Feedback Loop:** Establish an effective feedback loop where maintainers can
+provide insights and guidance on contributions.
+- **Decision-Making:** Make collective decisions on the adoption of new features,
+changes, and improvements based on technical merit.
+
+This process aligns with our commitment to open-source principles, ensuring that
+technical discussions are inclusive, transparent, and beneficial for the
+entire Keptn community.
+
+## Contributor guidance
+
+Contributors proposing new features are encouraged to participate in
+[refinement meetings](https://community.cncf.io/keptn-community/) related
+to their contributions.
+This provides valuable insights into ongoing technical discussions and
+aligns their efforts with the broader project vision.
+Follow these steps:
+
+1. Review the project's [contributing guide](../docs/contrib-guidelines-docs.md)
+for information on upcoming refinement meetings.
+2. Attend relevant refinement meetings to present and discuss proposed features.
+3. Actively engage in technical discussions, seeking feedback and guidance from
+maintainers.
+4. Iteratively refine contributions based on insights gained during meetings.
+
+By following this process, contributors contribute not only code but also valuable
+perspectives and insights, fostering a collaborative and innovative
+community environment.
+
+## Outcome
+
+- A shared understanding among maintainers and contributors regarding the technical
+landscape of ongoing developments.
+- Improved contributions through collaborative discussions and iterative refinement.
+- Documented decisions and action items for future reference in project documentation.
+This refined process aligns with our commitment to open-source principles, ensuring
+transparency, inclusivity, and technical excellence within the Keptn community.
```

Comment:
"inclusivity" is a legitimate word so take the steps to add it to our dictionary.  Instructions are in the PR I referenced.